jupyter_core/command.py

```diff
@@ -49,6 +49,7 @@ def list_subcommands():
     Returns a list of jupyter's subcommand names, without the `jupyter-` prefix.
     Nested children (e.g. jupyter-sub-subsub) are not included.
```

Comment:
Then should we rearrange `install-nbextension`? Or do we actually want to have an `install` subcommand namespace?